wp-content/civi-extensions/goonjcustom/Civi/InstitutionCollectionCampService.php

```diff
@@ -153,4 +155,56 @@ public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
       ->execute();
   }
 
+  /**
+   *
+   */
+  private static function isViewingInstituteCollectionCamp($tabsetName, $context) {
+    if ($tabsetName !== 'civicrm/eck/entity' || empty($context) || $context['entity_type']['name'] !== self::ENTITY_NAME) {
+      return FALSE;
+    }
+
+    $entityId = $context['entity_id'];
+
+    $entity = EckEntity::get(self::ENTITY_NAME, TRUE)
+      ->addWhere('id', '=', $entityId)
+      ->execute()->single();
+
+    $entitySubtypeValue = $entity['subtype'];
+
+    $subtypeId = self::getSubtypeId();
+
+    return (int) $entitySubtypeValue === $subtypeId;
+  }
+
+  /**
+   *
+   */
+  public static function InstitutionCollectionCampTabset($tabsetName, &$tabs, $context) {
```

Comment:
It violates PSR-1 naming conventions
```suggestion
  public static function institutionCollectionCampTabset($tabsetName, &$tabs, $context) {
```